product-service/tests/getProductsById.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { Product } from '@models/product.model';
+import { getById } from '../src/services/product.service';
+
+describe('getById', () => {
```

Comment:
According to the AC of the task _Lambda handlers_ should be covered by tests, these are not exactly them.
I believe it's assumed that `fuctions/getProductById` and `fuctions/getProductsList` should be tested